flake.nix

```diff
@@ -2,7 +2,13 @@
   description = "NixOS systems for internal services";
 
   inputs = {
-    nixpkgs.url = "github:serokell/nixpkgs";
+    # https://github.com/serokell/nixpkgs/pull/46
```

Comment:
Btw, it's not my business, but I see it's approved, so can it be merged and can master be used here?